src/button/index.tsx

```diff
@@ -40,6 +40,7 @@ const Button = React.forwardRef(
       fullWidth,
       form,
       i18nStrings,
+      nativeAttributes,
```

Comment:
Should we add them to metrics metadata? E.g. `nativeAttributesUsed: boolean`

Reply:
This is only exposed for core, and core shouldn't be used anywhere where metrics metadata is actually gathered